packages/cli/src/PublicApi/v1/handlers/executions/executions.handler.ts

```diff
@@ -33,13 +27,11 @@ export = {
 			// look for the execution on the workflow the user owns
 			const execution = await getExecutionInWorkflows(id, sharedWorkflowsIds, false);
 
-			if (!execution) {
+			if (!execution?.id) {
```

Comment:
is it now possible to have an execution here without an id?

Reply:
Indeed, after https://github.com/n8n-io/n8n/pull/7069 this ID should always be present. Updated.